pages/common/k3d.md

```diff
@@ -0,0 +1,24 @@
+# k3d
+
+> K3d is a wrapper CLI that helps you to easily create k3s clusters inside docker.
+> More information: <https://k3d.io/>.
+
+- Create a cluster:
+
+`k3d cluster create {{cluster_name}}`
+
+- Delete a cluster:
+
+`k3d cluster delete {{cluster_name}}`
+
+- Create a new containerized k3s node:
+
+`k3d node create {{node_name}}`
+
+- Import a image from docker into k3d cluster:
```

Comment:
```suggestion
- Import an image from docker into k3d [c]luster:
```
We prefer long arguments but when we use short single-character arguments we use mnemonics ([]) to show the long arguments.